src/xclim/indices/_hydrology.py

```diff
@@ -2,9 +2,13 @@
 
 from __future__ import annotations
 
+from functools import partial
+
 import numpy as np
+import pandas as pd
+import pymannkendall as mk  # FIXME: This is a new dependency. Is this needed at the scope of the package?
```

Comment:
This dependency supports a single function. Would it be alright to make this optionally-installed?